conf/messages.cy

```diff
@@ -118,10 +120,18 @@ pensions.pensionSummary.incomeFromPensions = Income from pensions
 pensions.pensionSummary.pensionAnnualAllowance = Pension annual allowance
 pensions.pensionSummary.pensionLifetimeAllowance = Pension lifetime allowance
 pensions.pensionSummary.unauthorisedPaymentsFromPensions = Unauthorised payments from pensions
-pensions.pensionSummary.paymentsToOverseasPensions = Payments into overseas pensions
+pensions.pensionSummary.paymentsToOverseasPensions = Taliadau i bensiynau tramor
 
 
 ########    PAYMENTS INTO PENSIONS    ########
+pensions.paymentsIntoOverseasPensions.statusPage.error.agent = Dewiswch ‘Iawn’ os gwnaeth eich cleient neu ei gyflogwr dalu i mewn i gynllun pensiwn tramor
+pensions.paymentsIntoOverseasPensions.statusPage.error.individual = Dewiswch ‘Iawn’ os gwnaethoch chi neu''ch cyflogwr dalu i mewn i gynllun pensiwn tramor
```

Comment:
The word `statusPage` could be removed to match the other properties. So each property would start with `pensions.paymentsIntoOverseasPensions.`

Reply:
Agree, will remove